lib/livebook/hubs/team_client.ex

```diff
@@ -837,25 +837,25 @@ defmodule Livebook.Hubs.TeamClient do
     put_in(state.hub, hub)
   end
 
-  defp put_billing_status(hub, %LivebookProto.BillingStatus{} = status) do
+  # TODO: Remove when Billing is public
+  defp put_billing_status(hub, nil = _status) do
```

Comment:
@wojtekmach I changed this here to match in `(_, nil)`, because now we're properly sending the billing status as nil from Teams, given one the last PR you merged.

This what Livebook receives after connecting to Teams:

```elixir
%LivebookProto.UserConnected{
  name: "radar-test", 
  secrets: [], 
  file_systems: [], 
  deployment_groups: [],
  app_deployments: [], 
  agents: [], 
  billing_status: nil,  # billing_status is nil,
  __unknown_fields__: []
}
```